readme.md

```diff
@@ -9,67 +9,100 @@ A Node.js client for [Segment](https://segment.com) — The hassle-free way to i
 
-3. **Send data from anywhere**. Send Segment data from any device, and we'll transform and send it on to any tool.
+```js
+const Analytics = require('@dreamdata/analytics');
 
-4. **Query your data in SQL**. Slice, dice, and analyze your data in detail with Segment SQL. We'll transform and load your customer behavioral data directly from your apps into Amazon Redshift, Google BigQuery, or Postgres. Save weeks of engineering time by not having to invent your own data warehouse and ETL pipeline.
+const client = new Analytics('write key');
 
-    For example, you can capture data on any app:
-    ```js
-    analytics.track({ userId: '019mr8mf4r', event: 'Order Completed', properties: { price: 99.84 }});
-    ```
-    Then, query the resulting data in SQL:
-    ```sql
-    select * from app.order_completed
-    order by price desc
-    ```
+client.identify({
+    userId: '019mr8mf4r',
+    traits: {
+        email: "name1@email.com",
+        name: "Name",
+        age: 25
+    }
+})
 
-### 🚀 Startup Program
-<div align="center">
-  <a href="https://segment.com/startups"><img src="https://user-images.githubusercontent.com/16131737/53128952-08d3d400-351b-11e9-9730-7da35adda781.png" /></a>
-</div>
-If you are part of a new startup  (&lt;$5M raised, &lt;2 years since founding), we just launched a new startup program for you. You can get a Segment Team plan  (up to <b>$25,000 value</b> in Segment credits) for free up to 2 years — <a href="https://segment.com/startups/">apply here</a>!
+client.track({
+    event: 'Song Played',
+    userId: '019mr8mf4r',
+    properties: {
+        name: "The song title 1",
+        artist: "The artist 1"
+    }
+})
+
+client.identify({
+    userId: '971mj8mk7p',
+    traits: {
+        email: "name2@email.com",
+        name: "Name2",
+        age: 26
+    }
+})
 
-## Installation
+client.track({
+    event: 'Song Played',
+    userId: '971mj8mk7p',
+    properties: {
+        name: "The song title 2",
+        artist: "The artist 2"
+    }
+})
+```
 
-```bash
-$ npm install analytics-node
+## Use Batches
 ```
+Our libraries are built to support high performance environments. That means it is safe to use our Node library on a web server that’s serving hundreds of requests per second.
 
+Every method you call does not result in an HTTP request, but is queued in memory instead. Messages are then flushed in batch in the background, which allows for much faster operation.
 
-## Usage
+By default, our library will flush:
 
-```js
-const Analytics = require('analytics-node');
+The very first time it gets a message.
+Every 20 messages (controlled by options.flushAt).
+If 10 seconds has passed since the last flush (controlled by options.flushInterval)
+There is a maximum of 500KB per batch request and 32KB per call.
```

Comment:
this reads a bit choppy, did we steal it or pick and choose some of it?

Reply:
I stole it from the segment documentation. Its just the part about how the batches work. What is the default values and how do you change them. I thought it made sense to have that because we are using the batch endpoint.

But maybe I should have added them in another commit? I just forced pushed it on the commit "Update documentation"

Reply:
Made some changes to this. @paddie